build.sbt

```diff
@@ -13,9 +13,7 @@ lazy val root = (project in file("."))
     majorVersion := 0,
     DefaultBuildSettings.scalaSettings,
     DefaultBuildSettings.defaultSettings(),
-    scalaVersion := "2.13.12",
-    // To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always,
+    scalaVersion := "2.13.13",
```

Comment:
extract this to global level like this `ThisBuild / scalaVersion := "2.13.13"` per sbt plugin